gov2/cross_service/wordfreq/README.md

```diff
@@ -0,0 +1,131 @@
+# Word Frequency Counting example
+
+# Purpose
+
+This example demonstrates using mutliple services together to create a word frequency counting service. It demonstrates:
+
+* Using the CDK for Go to deploy and manage an environment
+* Using S3 event notifications to respond to events
+* Using SQS to deliver messages to endpoints
+* Using ECS Fargate to horizontally scale processing based on load
```

Comment:
User guide usage is:
Amazon Elastic Container Service (Amazon ECS) for AWS Fargate